src/commands/verify/subcommands/submit/verify-submit.handler.ts

```diff
@@ -58,7 +58,7 @@ export const handleVerifySubmit = async (
 		if (yearRole && year) {
 			await setYearRoles(user, yearRole, interaction.guild);
 			await handleChannelAlias(mapYearToAlias(year), interaction, joinChannel);
-		}
+		} else setExternRole(user, interaction.guild);
```

Comment:
I can't see why we would want to make this assumption really.
Is it easier to find all of this roles instead of looking for someone who is verified but does not have a year?

Anyway. It might make sense since we have years in hodis for everyone (or almost everyone I would assume) from 2017.